comment/views.py

```diff
@@ -0,0 +1,86 @@
+from django.shortcuts import render
+
+# Create your views here.
+from rest_framework import status
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .models import Comment
+from .serializers import CommentSerializer
+from post.models import Post
+
+
+class CommentListView(APIView):
+	### 1 ###
+  def get(self, request):
+
+    post_id = request.GET.get('post')
+    if not post_id:
+      return Response({"detail": "missing fields ['post']"}, status=status.HTTP_400_BAD_REQUEST)
+    
+    comments = Comment.objects.filter(post_id=post_id)
+    if not Post.objects.filter(id=post_id).exists():
+      return Response({"detail" : "Not found."}, status=status.HTTP_404_NOT_FOUND)
+  
+    serializer = CommentSerializer(instance=comments, many=True)
+    return Response(serializer.data, status=status.HTTP_200_OK)
+
+	### 2 ###
+  def post(self, request):
+    if not request.user.is_authenticated:
+      return Response({"detail": "Authentication credentials not provided"}, status=status.HTTP_401_UNAUTHORIZED)
+  
+    author = request.user
+    post_id = request.data.get('post')
+    content = request.data.get('content')
+
+    if not author.is_authenticated:
+        return Response({"detail": "Authentication credentials not provided"}, status=status.HTTP_401_UNAUTHORIZED)
+
+    if not content or not post_id:
+      return Response({"detail": "missing fields ['post', 'content']"}, status=status.HTTP_400_BAD_REQUEST)
+    
+    if not Post.objects.filter(id=post_id).exists():
+      return Response({"detail" : "Not found."}, status=status.HTTP_404_NOT_FOUND)
+
+    comment = Comment.objects.create(author=author, post_id=post_id, content=content)
+    serializer = CommentSerializer(comment)
+    return Response(serializer.data, status=status.HTTP_201_CREATED)
+
+  ##Only for revision
+  ##3
+  def put(self, request, comment_id):
+
+    comment = Comment.objects.get(id=comment_id)
+
+    if not request.data.get('content'):
+      return Response({"detail": "missing fields ['content']"}, status=status.HTTP_400_BAD_REQUEST)
+    
+    if not request.user.is_authenticated:
+      return Response({"detail": "Authentication credentials not provided"}, status=status.HTTP_401_UNAUTHORIZED)
+
+    if request.user != comment.author:
+        return Response({"detail": "Permission denied"}, status=status.HTTP_401_UNAUTHORIZED)
+    
+    if not comment:
+        return Response({"detail": "Not found."}, status=status.HTTP_404_NOT_FOUND)
```

Comment:
그래서 해당 Id를 가진 comment가 없더라도 이 if문까지 내려올 수가 없습니다.. ㅠㅠ
그래서 아래와 같이 고쳐주는 것이 좋습니다!
```suggestion
    try:
        Comment.objects.get(id=comment_id)
    except:
        return Response({"detail": "Not found."}, status=status.HTTP_404_NOT_FOUND)
```

이 코드는 여기가 아니라 위쪽에 들어가야겠죠!